src/__tests__/downshift.get-input-props.js

```diff
@@ -583,6 +583,25 @@ test(`getInputProps doesn't include event handlers when disabled is passed (for
   }
 })
 
+test('highlightedIndex is reset to defaultHighlightedIndex when inputValue changes', () => {
+  const props = {defaultHighlightedIndex: 0}
+  const {childrenSpy, arrowDownInput, changeInputValue} = renderDownshift({
+    props,
+  })
+
+  childrenSpy.mockClear()
+
+  arrowDownInput() // highlightedIndex = 1
+
+  changeInputValue('c')
```

Comment:
let's input `r` instead, to make sure the number of resulting items is more than one (in this case it will be 4). that will actually test that the highlight is going to the right index, not just because there was only one option (if you type `c` you will get only one option)

Reply:
good point